mutation/index.ts

```diff
@@ -0,0 +1,132 @@
+import { useCallback, useRef } from 'react'
+import useSWR, {
+  useSWRConfig,
+  Middleware,
+  Key,
+  Fetcher,
+  SWRResponse
+} from 'swr'
+
+import { serialize } from '../src/utils/serialize'
```

Comment:
Non-blocking one

we can import `unstable_serialize` here from swr instead of importing source code from src folder.
currently the type files (.d.ts) aren't able to be bundled. so it will emit some useless typing files in dist which has the similar structure of source code like `src/utils/serialize.d.ts`.

we can look for another way to optimize those imports in the future

Reply:
Just realized that we can't use `unstable_serialize`, since here we are using the `args` as well (but `unstable_serialize` only gives the key):

```js
const [serializedKey, args] = serialize(keyRef.current)
```